frontend/src/employee-frontend/components/UnitPage.tsx

```diff
@@ -147,9 +147,13 @@ export default React.memo(function UnitPageWrapper() {
   )
 })
 
-export const NotificationCounter = styled.span`
-  height: 21px;
-  padding: 4px 8px;
+export const NotificationCounter = styled.div`
+  display: inline-block;
+  width: 24px;
+  height: 24px;
+
+  text-align: center;
+  padding: 3px 2px;
```

Comment:
Using `display: inline-flex; justify-content: center; align-items: center;` has a more consistent result in my opinion and you won't need the `padding` then either.